src/test/java/hudson/plugins/build_timeout/impl/ElasticTimeOutStrategyTest.java

```diff
@@ -40,15 +39,15 @@ public void percentageWithTwoBuilds() throws Exception {
 
         Build b = new Build(new Build(20 * MINUTES, SUCCESS, new Build(40 * MINUTES, SUCCESS)));
 
-        assertEquals("Timeout should be 200% of the average of 20 and 40", 60 * MINUTES, strategy.getTimeOut(b,null));
+        assertEquals( 60 * MINUTES, strategy.getTimeOut(b,null),"Timeout should be 200% of the average of 20 and 40");
     }
 
     @Test
     public void percentageWithNoBuilds() throws Exception {
         BuildTimeOutStrategy strategy = new ElasticTimeOutStrategy(200, 90, 3);
 
         Build b = new Build(null);
-        assertEquals("Timeout should be the elastic default.", 90 * MINUTES, strategy.getTimeOut(b,null));
+        assertEquals( 90 * MINUTES, strategy.getTimeOut(b,null),"Timeout should be the elastic default.");
```

Comment:
Same here

Reply:
Now i can see build succeded after removing space. Thanks @krisstern for pointing out!

Reply:
Great that works @baruKreddy